hometask7.py

```diff
@@ -0,0 +1,111 @@
+# Task 1
+import json
+
+
+try:
+    with open('json_phonebook.json', 'r') as file:
+        data = file.read()
+        phonebook = json.loads(data)
+except FileNotFoundError:
+    # if file does not exist
+    phonebook = {}
+
+while True:
+    user_input = input('Enter a command:')
+    split_input = user_input.split()
+    command = split_input[0]
+
+
+    if command == 'stats':
+        print(len(phonebook))
+
+
+    elif command == 'add':
+        name = input('Enter a name:')
+        if not name in phonebook.keys():
+            number = input('Enter a number:')
+            phonebook[name] = number
+            with open('json_phonebook.json', 'a') as file:
```

Comment:
If you appen (mod `a`) data to the file you json structure will become invalid. Use `w+` modifier here instead